skimage/segmentation/tests/test_random_walker.py

```diff
@@ -430,10 +432,50 @@ def test_isolated_seeds():
     mask[6, 6] = 1
 
     # Test that no error is raised, and that labels of isolated seeds are OK
-    with expected_warnings([NUMPY_MATRIX_WARNING]):
+    with expected_warnings([NUMPY_MATRIX_WARNING,
+                            'The probability range is outside']):
         res = random_walker(a, mask)
     assert res[1, 1] == 1
-    with expected_warnings([NUMPY_MATRIX_WARNING]):
+    with expected_warnings([NUMPY_MATRIX_WARNING,
+                            'The probability range is outside']):
+        res = random_walker(a, mask, return_full_prob=True)
+    assert res[0, 1, 1] == 1
+    assert res[1, 1, 1] == 0
+
+
+def test_prob_tol():
+    # Turn warnings into errors
+    warnings.simplefilter('error')
```

Comment:
I think this is good for debugging, but I don't think it should persist through the test suite.

Reply:
I'm just afraid that otherwise, we miss a failure. What's your suggestion?

Reply:
ah good catch. Anyway uncaught warnings make the test suite fail.

Reply:
@emmanuelle shall I understand that you want me also to remove this line?

On my machine, the test was not failing without this line. Is there another configuration in the CI that I missed? I remember that warnings were supposed to fail at some point, but I can't remember how this is set.

Reply:
Found it. The same line is in setup_test()

Reply:
@sciunto so are you going to remove it?? =) At which point you will have my approval. =)

Reply:
Now that I understand, yes :D

Reply:
I just pushed the commit